### PR TITLE
Make the makefile work for those that have a default python of v3

### PR DESCRIPTION
**Pull request by calvinhp**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 
 install:
-	python2.7 -m virtualenv .
```
Comment (whit537) [inline Makefile]: `python2.7` points to Python 3 on your system? :-P

**Comment by whit537** [comment]
I don't quite understand the need for this, but it does work for me. 👍

```
$ python --version
Python 3.5.2
$ make
python -m virtualenv --python python2.7 .
Running virtualenv with interpreter /usr/local/bin/python2.7
New python executable in /Users/whit537/workbench/capitalone/cloud-custodian/bin/python2.7
Also creating executable in /Users/whit537/workbench/capitalone/cloud-custodian/bin/python
Installing setuptools, pip, wheel...done.
source bin/activate && pip install -r requirements.txt
Collecting boto3>=1.4.0 (from -r requirements.txt (line 1))
  Downloading boto3-1.4.3-py2.py3-none-any.whl (127kB)
    100% |████████████████████████████████| 133kB 2.2MB/s 

[...]

$ Finished processing dependencies for c7n==0.8.21.2
$ . bin/activate
(cloud-custodian) $ python --version
Python 2.7.12
(cloud-custodian) $
```

**Comment by kapilt** [comment]
um.. where do you get a default python3 unless custom installing, no os or distro would do that, there's way too much breakage in the wild to change the default to `python` to an incompatible implementation.

that said, this looks fine, thanks.

File: Makefile (hunk 1 above)
Comment (calvinhp) [inline Makefile]: I have `virtualenv` only installed into my Python 3.5 installation and wanted to be explicit about using Python 2.7 for this project. This refactor makes it work correctly either way.

**Pull request merged**